src/App.tsx

```diff
@@ -1,5 +1,49 @@
+import colors from './colors.json';
+
+function ColorObject({ colorInfo }) {
+    return (
+        <div
+            style={{
+                color: `#${colorInfo.hex}`,
+            }}
+        >
+            {colorInfo.name}
+        </div>
+    );
+}
+
+function ColorObjectWithComp({ colorInfo }) {
+    return (
+        <div style={{ display: 'flex', gap: '12px' }}>
+            <div style={{ width: '128px' }}>
+                <ColorObject colorInfo={colorInfo} />
+            </div>
+            <div>Complimentary: </div>
+            <div style={{ display: 'flex', gap: '12px' }}>
+                {colorInfo.comp.map(i => {
+                    return (
+                        <div key={i.hex}>
+                            <ColorObject colorInfo={i} />
+                        </div>
+                    );
+                })}
+            </div>
+        </div>
+    );
+}
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    return (
+        <div>
+            {colors.map(i => {
```

Comment:
Some of the colors are black, and so the text becomes black, which is difficult to read against the black background. This is the risk of using text color to demonstrate the color.

Reply:
You are not displaying the hex codes anywhere - it would be nice to see them somewhere, or have them available through some user interaction. Maybe they would show when you hover, maybe you could click on the color and it would copy the hex code to your clipboard.